internal/plugins/ansible/v1/init.go

```diff
@@ -128,6 +129,13 @@ func (p *initPlugin) runPhase2() error {
 	if err := scorecard.RunInit(p.config); err != nil {
 		return err
 	}
+
+	if p.doCreateAPI {
+		if err := p.apiPlugin.runPhase2(); err != nil {
+			return err
+		}
+	}
```

Comment:
Remind me why we don't just put the `runPhase2()` methods on the init and create api scaffolders, and then just call them when we're doing the other scaffolding?

Is this a separation of concerns thing where we'll keep these scaffolds in operator-sdk even after we split out the "core" plugin logic in separate repos?

Reply:
Every plugin called in `runPhase2()` will be a phase 2 plugin, i.e. not be part of scaffolding in any phase 1 plugin, so it makes sense to treat them as separate objects and call them only after `{Plugin}.Run()` completes. There isn't a clear picture as to where these plugins will live.

Reply:
IMO keep the "plugin phase 2" centralized with the comment `// SDK phase 2 plugins.` as suggested in #3658 clarify better what it is doing, when it needs to be done (after the scaffold), and what will be `SDK phase 2 plugins.`.